src/test/java/io/vlingo/http/resource/sse/SseFeedTest.java

```diff
@@ -33,11 +32,11 @@ public void testThatFeedFeedsOneSubscriber() {
 
     final SseSubscriber subscriber = new SseSubscriber("all", client, "ABC123", "42");
 
-    context.channel.untilRespondWith = TestUntil.happenings(1);
+    context.channel.expectRespondWith(1);
 
     feed.to(Arrays.asList(subscriber));
 
-    context.channel.untilRespondWith.completes();
+    context.channel.untilCompletes();
```

Comment:
Eliminate this line and replace with:
`assertEquals(1, (int) respondWithSafely.readFrom("foo"));`